src/directives/templates/mmsRefList.html

```diff
@@ -59,7 +59,7 @@
                     <ul class="dropdown-menu pull-right" uib-dropdown-menu role="menu" aria-labelledby="single-button">
                         <li role="menuitem" style="padding:10px;">
                             <button class="btn btn-default btn-sm" ng-class="{'disabled': !docEditable || !runCleared}" ng-click="docMergeAction(branch)">
-                                Apply<i ng-show="!runCleared" class="fa fa-spin fa-spinner"></i>
+                                Pull In<i ng-show="!runCleared" class="fa fa-spin fa-spinner"></i>
```

Comment:
@ellenvanwyk you decided to use "pull in" for doc merge as well?

Reply:
Yeah I iterated and it tested better